src/main/java/io/github/eocqrs/kafka/parameters/KfObjMapParams.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2022 Aliaksei Bialiauski, EO-CQRS
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.github.eocqrs.kafka.parameters;
+
+import io.github.eocqrs.kafka.Params;
+import io.github.eocqrs.kafka.ParamsAttr;
+import lombok.RequiredArgsConstructor;
+import org.cactoos.Scalar;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * @todo #147:30m/DEV /Tests for {@link io.github.eocqrs.kafka.parameters.KfObjMapParams}.
+ * We have to write unit tests.
+ */
+/**
+ * It converts a `Params` object into a `Map<String, Object>`.
+ *
+ * @author Ivan Ivanchuk (l3r8y@duck.com)
+ * @since 0.0.2
+ */
+@RequiredArgsConstructor
+final class KfObjMapParams implements Scalar<Map<String, Object>> {
```

Comment:
@l3r8yJ KfObjMapParams name is too complex.
MapParams can be more simplistic one